valet-key/ValetKey.Web/FileServices.cs

```diff
@@ -33,9 +33,12 @@ private async Task<StorageEntitySas> GetSharedAccessReferenceForUploadAsync(Blob
         {
             var blobServiceClient = blobContainerClient.GetParentBlobServiceClient();
             var blobClient = blobContainerClient.GetBlockBlobClient(blobName);
+            
+            var now = DateTimeOffset.UtcNow;
+            var startTime = now.AddMinutes(-1);
```

Comment:
Let's just keep it at the +/- 3 like it was before to help with clock drift.
```suggestion
            var startTime = now.AddMinutes(-3);
```